tools/Config.mk

```diff
@@ -268,6 +272,10 @@ else
   INCDIR ?= "$(TOPDIR)/tools/incdir$(HOSTEXEEXT)"
 endif
 
+ifeq ($(CONFIG_ARCH_TOOLCHAIN_TASKING),)
+  INCDIR += -s
```

Comment:
but' it's better to move into incdir.sh like other compiler

Reply:
Incdir has multiple versions, c and shell. so，do we need to apply the similar change for all versions?

Reply:
I think apply changes here would be more reasonable

Reply:
but, all compiler special options is handled in incdir.sh

Reply:
ok. done